### PR TITLE
Feat/card

### PR DESCRIPTION
**Pull request by bhavyaj12**
Added 7 card variants

1. Text only card 
2. text shadow card
3. horizontal card
4. vertical card
5. card with badge
6. card with dismiss
7. card with overlay

card

### CODE REVIEW

**Review by masraniJay — APPROVED** [review]
Looks good.

**Review by arulk8 — APPROVED** [review]

**Review by arulk8 — APPROVED** [review]
Good to go. UI is nice

File: css/card.css [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  border: 2px solid var(--grey-bg);
+  margin: var(--size-1) var(--size-1);
+}
+
+.discount {
+  color: var(--disc-green);
+}
+
+.card-text-only h2 {
+  margin: 0 0 1rem 0;
```
Comment (Prabhav-Sharma) [inline css/card.css:24]: You can use global variables for spacing here as well.

File: css/button.css [hunk 2]
```diff
@@ -43,7 +49,7 @@
     min-height: max-content;
 }
 
-.button-secondary.btn-solid{
+.button-secondary.btn-solid {
     background-color: var(--secondary-color);
     color: black;
```
Comment (amirnathani9) [inline css/button.css:54]: Use the var for color. Don't write directly color name.

File: css/card.css [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+  border-radius: 0.3rem;
+  font-weight: 500;
+  position: absolute;
+  top: 5px;
+  left: 0.1px;
+}
+
+.card .btn-close {
+  position: absolute;
+  background-color: var(--white-bg);
```
Comment (amirnathani9) [inline css/card.css:143]: Instead of giving the color name to a variable, you can use `light-bg` or `bg-color-1,bg-color2.`

File: css/input.css [hunk 4]
```diff
@@ -14,15 +14,15 @@
...
 }
 
 .input-label {
     padding: var(--size-0-5) 0;
     display: block;
 }
 
-.input-field:disabled{
+.input-field:disabled {
     background-color: var(--grey-bg);
```
Comment (amirnathani9) [inline css/input.css:26]: the same suggestion here for color variable naming

**Comment by amirnathani9** [comment]
Your Ui looks good. 
One suggestion is in the card component. Vertical card height is too much it cannot come in one page. Try to decrease the size of the card.
And the second suggestion does not edit your other component in the component branch. Like in card component don't edit avatar,alert,etc.

File: css/input.css (hunk 4 above)
Comment (bhavyaj12) [reply css/input.css:26]: Thanks a lot !

**Pull request merged**